numpy/__init__.pyi

```diff
@@ -154,8 +154,324 @@ __all__ = [
     "var",
 ]
 
-# TODO: remove when the full numpy namespace is defined
-def __getattr__(name: str) -> Any: ...
+DataSource: Any
+False_: Any
+MachAr: Any
+ScalarType: Any
+True_: Any
+UFUNC_PYVALS_NAME: Any
+abs: Any
```

Comment:
```suggestion
```
Seems like we already added an annotation for this in https://github.com/numpy/numpy/pull/16618.

Reply:
Thank you. Done.